ajax/libs/scrolldir/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "scrolldir",
+  "filename":"scrolldir.min.js",
+  "version": "1.2.2",
+  "description": "Vertical scroll direction in CSS",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/dollarshaveclub/scrolldir.git"
+  },
+  "keywords": [
+    "jquery",
+    "scroll",
+    "intent",
+    "scrollTop",
+    "position",
+    "javascript",
+    "window"
+  ],
+  "author": "Patrick Fisher <patrick@pwfisher.com>",
+  "license": "MIT",
+  "homepage": "https://dollarshaveclub.github.io/scrolldir/",
+  "npmName": "scrolldir",
+  "npmFileMap": [
+    {
+      "basePath": "src",
```

Comment:
we only host pre-built files for browser, which should be the files under `dist`.